newdle/client/src/components/creation/timeslots/Timeline.module.scss

```diff
@@ -3,8 +3,7 @@
 $row-height: 50px;
 $label-width: 180px;
 
-.timeline {
-  margin: 4px;
+@mixin timeline-all {
```

Comment:
Why set this as a @mixin? Why not including it together with `.timeline` selector?

Reply:
not necessary any more, I remove it with the next push